stations.csv

```diff
@@ -62012,3 +62012,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 66143;Äs;as;;;59.15167;16.10778;;SE;Europe/Stockholm;t;f;f;f;t;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66143;
 66144;Örnsköldsvik Fjällräven Center;ornskoldsvik-fjallraven-center;;;63.28417;18.72444;35819;SE;Europe/Stockholm;f;f;f;f;t;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66144;
 66145;Östersund Central;ostersund-central;;;63.17028;14.63778;35768;SE;Europe/Stockholm;f;f;f;f;t;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;エステルスンド;외스테르순드;;;Эстерсунд;;;厄斯特松德;urn:trainline:public:nloc:csv66145;
+66146;Cassino Abbazia;cassino-abbazia;8313956;;41.2924;13.4850;19827;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8313956;f;;f;;f;;f;;f;;f;;f;;f;8313956;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;カッシーノ;;;;Кассино;;;;urn:trainline:public:nloc:csv66146;
```

Comment:
Please add a new line at the end of the file

Reply:
Done!